### PR TITLE
Run cargo-make CI on pushes to master

### PR DESCRIPTION
**Pull request by adamgreig**

### CODE REVIEW

**Comment by adamgreig** [comment]
@burrbull, any idea what's going wrong with the cargo-make CI ([here](https://github.com/stm32-rs/stm32-rs/actions/runs/5108291868/jobs/9181968336?pr=843))? The same commands seem to work fine locally for me, and I can't work out how to diagnose exactly what file it's missing as they all seem to exist...

**Comment by burrbull** [comment]
Line 24 of script.
dir_name?

File: Makefile.toml [hunk 1]
```diff
@@ -108,8 +108,14 @@ for i in ${handle}
...
+        echo "modpath: ${modpath}, dir_name: ${dir_name}"
         exec form -i ${modpath} -o ${dir_name}
+        ls ${dir_name}
         rm ${modpath}
+        echo "dir_name: ${dir_name}"
+        echo "modpath: ${modpath}"
+        echo "ls dir_name:"
+        ls ${dir_name}
         cp ${dir_name}/lib.rs ${modpath}
         rm ${dir_name}/lib.rs
```
Comment (burrbull) [inline Makefile.toml:120]: try to just move `lib.rs`
```suggestion
        echo "dir_name: ${dir_name}"
        echo "modpath: ${modpath}"
        echo "ls dir_name:"
        ls ${dir_name}
        mv ${dir_name}/lib.rs ${modpath}
```

**Comment by burrbull** [comment]
```
          ./scripts/tool_install.sh form
```

**Comment by burrbull** [comment]
duskscript points to incorrect line because it internally add lines with setting `env`.
It just could not find `form`

**Comment by adamgreig** [comment]
Whew, thank you!

**Pull request merged**